src/backend/web/graphql/queries.js

```diff
@@ -0,0 +1,99 @@
+const { gql } = require('apollo-server-express');
+
+const maxPostsPerPage = 30;
+
+module.exports.typeDefs = gql`
+  # 'Feed' matches our Feed type used with redis
+  type Feed {
+    name: String
+    url: String
+  }
+
+  # 'Post' matches our Post type used with redis
+  type Post {
+    author: String
+    title: String
+    html: String
+    text: String
+    published: String
+    updated: String
+    url: String
+    site: String
+    guid: String
+  }
+
+  # Queries to fetch data from redis
+  type Query {
+    getFeed(guid: ID!): Feed
+    getFeeds: [ID]
```

Comment:
Another question: what do GraphQL clients expect when requesting data: getting back IDs (like a REST API); or automatic rehydration with full Object data?  I don't have a lot of experience designing these APIs, so I'm unclear how we should model the data.  Should I be able to request a `feed` or an Array of `feeds` using essentially the same query?

Reply:
>  Should I be able to request a feed or an Array of feeds using essentially the same query?

Yes, that's essentially the idea.
Our REST API has `GET /post` and `GET /post/:guid`. The first endpoint returns an array of `guids` that can be used with the second one.

With GraphQL, `/grapql` returns any subfield(s) included in the query, so:

`GET /post` becomes
```
getPosts()
{
    author
    title
    html
    text
    published
    updated
    url
    site
    guid
}
```

and `GET /posts/:guid` becomes
```
getPosts()
{
    guid
}
```

All in one query. 
I added this query `getPost(guid: ID!): Post` to return a post for a given ID because `getPosts` needs some sort of filtering functionality added. I did a bit of research for this, but I thought it was better to try get basic functionality in and then tweak this part with future PRs.
Also, the way getPosts works right now is enough to get Telescope to work with Gatsby.